doc/certificate-handling.md

```diff
@@ -0,0 +1,12 @@
+# Kubemacpool webhook certificate handling
+
+Using admission webhooks forces the kubemacpool application to handle TLS CA and server
+certificates. This task is delegated to [kube-admission-webook](https://github.com/qinqon/kube-admission-webhook)
+library, which manages certificates rotation using a controller-runtime webhook server and a small cert manager.
+
+To customize how this rotation works some knobs from the library has being
+exposed as environment variables at kubemacpool pod:
+
+- `CA_ROTATE_INTERVAL`:  Expiration time for CA certificate, default is one year
+- `CA_OVERLAP_INTERVAL`:  Duration the previous to rotate CA certificate live with the new one, defaults to one year
```

Comment:
The description of CA_OVERLAP_INTERVAL is a bit hard to understand, is the default really one year?
The way I understand this intervals is that within this time both certificates are accepted.

Reply:
I personally don't get it either. @qinqon perhaps you can give a better explanation?

Reply:
During the overlap intevarl both CA certs are valid (if they are not expired), they library has one year duration by default https://github.com/qinqon/kube-admission-webhook/blob/master/pkg/certificate/options.go#L65-L82, but kubemacpool may be overriding it.

Reply:
Thanks for confirmation!

Reply:
@RamLavi Do you think you could rephrase the description?

Reply:
ok, so according to [this](https://github.com/qinqon/kube-admission-webhook/pull/28), I propose this:
```suggestion
- `CA_OVERLAP_INTERVAL`:  Duration where expired CA certificate can overlap with new one, in order to allow fluent CA rotation transitioning. defaults to one year
```

Reply:
Looks good